doc/unordered/unordered_node_map.adoc

```diff
@@ -537,6 +538,24 @@ and always calls `other.reset_stats()`.
 
 ---
 
+==== Move Constructor from concurrent_node_map
+
+```c++
+unordered_flat_map(concurrent_node_map<Key, T, Hash, Pred, Allocator>&& other);
```

Comment:
This should presumably say `unordered_node_map`?

Reply:
Good catch, fixed.